modules/quality_control/readme.md

```diff
@@ -0,0 +1,32 @@
+## Quality Control Module
+
+#### Purpose
+The purpose of this module is to ensure data has been uploaded into LORIS properly. LORIS employs double data entry as a quality control measure, and this module is designed to catch any inconsistencies.
+
+#### Intended Users
+This module is used by data entry staff, imaging scientists, analysts, coordinators, and clinicians.
+
+#### Scope
+The imaging tab allows you to view more specific data on uploaded image files, with many additional filtering options. The resulting table has a column for QC status, so you can see whether the upload has passed QC or not. The table indicates the location of each file as a clickable hyperlink. You can download the table in .csv format. 
+
+The Imaging Quality Control module is simply a tool to view where conflicts exist within your data. Once you’ve identified any issues, you’ll need to go elsewhere in LORIS to amend them. For example, the module may tell you that there’s a conflict with a double data entry point. From there, you would visit the Conflict Resolver module to address/fix the issue. 
+
+In other words, the The Imaging Quality Control module is designed to bring all quality control information into the same module, for ease of access.
+
+###### Permissions
```

Comment:
```suggestion
#### Permissions
```

I think Permissions and Interactions should be at the same level as Scope but I'm not positive.